### DISCUSSION TITLE
rerunning target linked to aws error message

### DISCUSSION THREAD

**Discussion by yonicd** [ropensci/targets#491] [Help]
I ran the example you have in the targets book

```r
library(targets)
my_bucket <- 'abc'
tar_option_set(resources = list(bucket = my_bucket))

write_mean <- function(data) {
  tmp <- tempfile()
  writeLines(as.character(mean(data)), tmp)
  tmp
}

list(
  tar_target(data, rnorm(5), format = "aws_qs"),
  tar_target(mean_file, write_mean(data), format = "aws_file")
)
```

i wanted to see the pipeline rerun after i deleted the objects in `my_bucket`

```r
> aws.s3::delete_object(bucket = my_bucket, object = "_targets/objects/data")
[1] TRUE
> aws.s3::delete_object(bucket = my_bucket, object = "_targets/objects/mean_file")
[1] TRUE
```

I can see it is running, but maybe the underlying calls when they get 404 could give back a more informative message? I understand what is happening underneath to know to ignore, but some might find these messages confusing.

```r
> tar_make()
Client error: (404) Not Found
* run target data
Client error: (404) Not Found
* run target mean_file
* end pipeline
```

**Comment by wlandau** [comment]
Unfortunately, I do not have control over those messages because they come from the `aws.s3` package.

**Reply by yonicd** [reply]
ok. thanks.
